### PR TITLE
Add nopStation ecommerce as technology

### PR DESCRIPTION
**Pull request by kazirahiv**
NopStation builts custom ecommerce solutions with custom integrations and custom built plugins based on tailored requirements on top of NopCommerce. Adding entry to detect sites built with nopCommerce and nopCommerce plugins

### CODE REVIEW

**Comment by nurbek91** [comment]
Hey mate, thanks for PR.
I'm not sure if nopStation can be added as a separate technology for ecommerce
examples:
https://nrbbazaar.com/
https://purplebybanu.com/
https://gasgasshopping.at/
https://motoshop24.ch/

On all the examples that I have tested it is either plugins or themes for nopCommerce
Probably it will be possible to add technology under category 19

File: src/technologies/n.json [hunk 1]
```diff
@@ -845,6 +845,20 @@
...
+  "nopStation": {
+    "cats": [
+      6
+    ],
+    "description": "One stop e-commerce solution with custom integrations and custom built plugins based on custom tailored requirements on top of NopCommerce",
+    "html": "<a[^>]+nop-station",
+    "icon": "nopStation.png",
+    "implies": "Microsoft ASP.NET",
+    "meta": {
+      "generator": "^nopStation$"
```
Comment (nurbek91) [inline src/technologies/n.json:857]: Do you have examples of sites with this fingerprint?

File: src/technologies/n.json [hunk 2]
```diff
@@ -845,6 +845,20 @@
     "oss": true,
     "website": "http://www.nopcommerce.com"
   },
+  "nopStation": {
+    "cats": [
+      6
+    ],
+    "description": "One stop e-commerce solution with custom integrations and custom built plugins based on custom tailored requirements on top of NopCommerce",
+    "html": "<a[^>]+nop-station",
```
Comment (nurbek91) [inline src/technologies/n.json:853]: It is not advisable to use html now
it can be replaced with dom

File: src/technologies/n.json [hunk 3]
```diff
@@ -845,6 +845,20 @@
...
+    "icon": "nopStation.png",
+    "implies": "Microsoft ASP.NET",
+    "meta": {
+      "generator": "^nopStation$"
+    },
+    "oss": true,
```
Comment (nurbek91) [inline src/technologies/n.json:859]: I think this field is invalid

File: src/technologies/n.json [hunk 4]
```diff
@@ -845,6 +845,20 @@
     "oss": true,
     "website": "http://www.nopcommerce.com"
   },
+  "nopStation": {
+    "cats": [
+      6
+    ],
+    "description": "One stop e-commerce solution with custom integrations and custom built plugins based on custom tailored requirements on top of NopCommerce",
```
Comment (nurbek91) [inline src/technologies/n.json:852]: nopStation is a one stop ecommerce solution with custom integrations and custom built plugins based on custom tailored requirements on top of nopCommerce.

**Review by nurbek91 — CHANGES_REQUESTED** [review]
.

**Comment by nurbek91** [comment]
nopStation
32x32

File: src/technologies/n.json (hunk 1 above)
Comment (kazirahiv) [reply src/technologies/n.json]: removed the meta tag

File: src/technologies/n.json (hunk 2 above)
Comment (kazirahiv) [reply src/technologies/n.json]: Replaced with dom

File: src/technologies/n.json (hunk 3 above)
Comment (kazirahiv) [reply src/technologies/n.json]: Removed the oss field

**Comment by kazirahiv** [comment]
@nurbek91 Hi, Made the changes, Added a new commit. Can you please check now ?

**Comment by kazirahiv** [comment]
> 

There is a lot of websites that we made but most of them has our plugins/integrations etc. So I added 6, 19 both.

**Comment by nurbek91** [comment]
> > 
> 
> There is a lot of websites that we made but most of them has our plugins/integrations etc. So I added 6, 19 both.

Hi, for 6 we try to define always one technology.
So we need to add the attribute:
`"exclude": "nopCommerce"`

But I'm not sure about this.

**Comment by kazirahiv** [comment]
> > > 
> > 
> > 
> > There is a lot of websites that we made but most of them has our plugins/integrations etc. So I added 6, 19 both.
> 
> Hi, for 6 we try to define always one technology. So we need to add the attribute: `"exclude": "nopCommerce"`
> 
> But I'm not sure about this.

Added the excludes property.

**Comment by nurbek91** [comment]
> > > > 
> > > 
> > > 
> > > There is a lot of websites that we made but most of them has our plugins/integrations etc. So I added 6, 19 both.
> > 
> > 
> > Hi, for 6 we try to define always one technology. So we need to add the attribute: `"exclude": "nopCommerce"`
> > But I'm not sure about this.
> 
> Added the excludes property.

But I'm not sure about this. It will probably still be correct to specify category 19 and add requires `"requires": "nopCommerce"`

**Comment by nurbek91** [comment]
Exactly how it will be right most likely can  only  @AliasIO  tell

**Comment by kazirahiv** [comment]
no problem, I made the changes to cat 19 and requires nopCommerce @nurbek91

**Comment by AliasIO** [comment]
Thank you!

**Comment by rockeynebhwani** [comment]
@AliasIO @kazirahiv - We are missing 'nopStation.png'.

**Pull request merged**